scripts/bigquery.js

```diff
@@ -0,0 +1,267 @@
+require('dotenv').config()
+const { BigQuery } = require('@google-cloud/bigquery')
+const { getMongo, connect } = require('../db')
+const { collections } = require('../services/constants')
+const { getAdvancedReports } = require('../services/sentry/analyticsRecorder')
+
+const REPORT_PUBLISHER_TO_ADUNIT_TABLE_NAME = 'reportPublisherToAdUnit'
+const REPORT_PUBLISHER_TO_COUNTRY_TABLE_NAME = 'reportPublisherToCountry'
+const BIGQUERY_RATE_LIMIT = 10 // There is a limit of ~ 2-10 min between delete and insert or changing schema
+const DATASET_NAME = process.env.DATASET_NAME || 'development'
+const options = {
+	keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+	projectId: process.env.GOOGLE_CLOUD_PROJECT
```

Comment:
I use `PROJECT_ID` and `DATASET` we should agree on the env names

Reply:
`GOOGLE_CLOUD_PROJECT` is the default used by BigQuery as `GOOGLE_APPLICATION_CREDENTIALS`. I have just set them explicitly. Can we stick to those names as Paunov is already using those for the market imports and it might get confusing?